3scale/porta
# THREESCALE-10074: Do not reveal that email doesn't exist on Forgot password form

**What this PR does / why we need it**:

When an invalid email is entered in the Forgot password? form, do not say that this email is not found.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-10074

**Verification steps** 

- Enter an email that doesn't have an associated user in the Forgot Password form
- Make sure the form just says "email has been sent"

**Special notes for your reviewer**:
